Azure/communication-ui-library-android
# [Refactor] UI change for Leave call confirm

## Purpose
UI change for Leave call confirm

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Join a call and press button leave call to verify if the leave call menu showing up on the bottom drawer.
